lib/rust/ensogl/core/src/lib.rs

```diff
@@ -25,6 +25,7 @@
 #![warn(unsafe_code)]
 // === Non-Standard Linter Configuration ===
 #![allow(clippy::option_map_unit_fn)]
+#![allow(clippy::precedence)]
```

Comment:
Should we add a comment why we allow this? (In case we want to revisit it at some point in the future)